spec/requests/admin/products_v3_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe "Admin::ProductsV3" do
+  include AuthenticationHelper
+
+  let(:user) { create(:user) }
+  let(:headers) { { Accept: "text/vnd.turbo-stream.html" } }
```

Comment:
Hmm, it would be nice if the turbo gem gave us shortcuts like it does for controller specs (`format: :turbo_stream`)